src/pass/estimate_flops.cc

```diff
@@ -56,6 +56,15 @@ void FLOPSEstimater::VisitExpr_(const LetNode* op) {
 }
 
 void FLOPSEstimater::VisitExpr_(const CallNode* call) {
+  if (call->op.as<OpNode>()) {
+    const Op& op = Downcast<Op>(call->op);
+    auto base_op = IsDialectOp(op) ? GetBaseOp(op) : op;
+    auto tvm_op = OpDialect::Lower(base_op, "tvm");
+    // skip this op if it does not have a TVM dialect
+    if (!tvm_op.defined()) {
+      return;
```

Comment:
Better to have a warning and leave an invalid value; otherwise the caller may encounter key error in `var_flops_map_`.

Meanwhile, I'm using infinity as the invalid value here. Another option is "0", but I'm afraid that this value will mass up with the ops with real 0 FLOPS.

```suggestion
      // #include <limits>
      var_flops_map_[curr_let_] = std::numeric_limits<float>::infinity();
      LOG(WARNING) << "Op " << base_op->name << " doesn't have TVM dialect, skip estimating FLOPS";
      return;
```

cc @zhouyuan1119

Reply:
That's a good point. I will modify accordingly. Also will compose a test.